data/Templates/Hl7v2/ADT_A01.liquid

```diff
@@ -2,7 +2,7 @@
     "resourceType": "Bundle",
     "type": "transaction",
     "entry": [
-        {% assign firstSegments = hl7v2Data | get_first_segments: 'PID|PD1|PV1|PV2|AVR|MSH' -%}
+        {% assign firstSegments = hl7v2Data | get_first_segments: 'PID|PD1|PV1|PV2|PR1|AVR|MSH' -%}
```

Comment:
It seems we can remove the PR1 here.

Reply:
Sure, it's a redundant while solve conflicts.